RecoBTag/Combined/src/CombinedMVAV2JetTagComputer.cc

```diff
@@ -99,6 +99,13 @@ float CombinedMVAV2JetTagComputer::discriminator(const JetTagComputer::TagInfoHe
   inputs["Jet_CSVIVF"]   = (*(computers[2]))( TagInfoHelper(ivfTagInfos) );
   inputs["Jet_SoftMu"]   = (*(computers[3]))( TagInfoHelper(smTagInfos) );
   inputs["Jet_SoftEl"]   = (*(computers[4]))( TagInfoHelper(seTagInfos) );
+  //std::cout << "JP=" << inputs["Jet_JP"]
```

Comment:
please drop the commented out code, or if you still need it, replace with LogDebug